src/app/main/dialog/dialog.component.html

```diff
@@ -3,41 +3,37 @@
     <!-- all about USER -->
     <!-- Create new user profile -->
     <div *ngSwitchCase="'createUser'">
-        <app-dialog-header [title]="'appLabels.form.user.title.profile' | translate"
-                           [subtitle]="'appLabels.form.user.title.new' | translate" (close)="dialogRef.close()">
+        <app-dialog-header [title]="'appLabels.form.user.title.profile' | translate" [subtitle]="'appLabels.form.user.title.new' | translate" (close)="dialogRef.close()">
         </app-dialog-header>
         <app-user-form [name]="data.name" [projectcode]="data.project" (closeDialog)="dialogRef.close()">
         </app-user-form>
     </div>
 
     <!-- Modify user data -->
     <div *ngSwitchCase="'editUser'">
-        <app-dialog-header [title]="data.name" [subtitle]="'appLabels.form.user.title.edit' | translate"
-                           (close)="dialogRef.close()"></app-dialog-header>
+        <app-dialog-header [title]="data.name" [subtitle]="'appLabels.form.user.title.edit' | translate" (close)="dialogRef.close()"></app-dialog-header>
         <app-user-form [username]="data.name" (closeDialog)="dialogRef.close()"></app-user-form>
     </div>
 
     <!-- Update password -->
     <div *ngSwitchCase="'editPassword'">
-        <app-dialog-header [title]="data.name" [subtitle]="'appLabels.form.user.title.password' | translate"
-                           (close)="dialogRef.close()"></app-dialog-header>
+        <app-dialog-header [title]="data.name" [subtitle]="'appLabels.form.user.title.password' | translate" (close)="dialogRef.close()"></app-dialog-header>
         <app-password-form [username]="data.name" (closeDialog)="dialogRef.close()"></app-password-form>
     </div>
 
     <!-- Delete user -->
     <div *ngSwitchCase="'deleteUser'">
-        <app-dialog-header [title]="data.name" [subtitle]="'Delete user'" (close)="dialogRef.close()">
+        <app-dialog-header [title]="data.name" [subtitle]="'Deactivate user'" (close)="dialogRef.close()">
```

Comment:
We had this discussion already and we wanted to use "suspend" instead of delete or deactivate. In the list of all users  we use already the term "Suspend user" and the title "Suspended users".

Reply:
I just saw your other changes about "suspend", and you changed all to "deactivate". Hm..., but if this was an issue by the test users, it's okay. Me personally don't like the term deactivate 😉

Reply:
s. https://github.com/dasch-swiss/knora-app/issues/174#issuecomment-556980210